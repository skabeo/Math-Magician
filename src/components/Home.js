import React from 'react';
import './styles.css';

const Home = () => (
  <div className="white homeCont">
    <h2 className="homeTitle">Welcome to our page</h2>
    <p className="homePara">Welcome to Math Magicians, the one-stop destination for all your math needs! Are you tired of staring at your calculator, trying to figure out basic calculations? Well, look no further because we have you covered! With Math Magicians, you can now perform basic calculations with just a few clicks.</p>
    <p className="homePara2"> And that&apos;s not all - we also have a surprise for you! How about some random quotes to keep you motivated and inspired throughout the day? Yes, you read that right! We&apos;ve combined the power of math and inspiration to create an unforgettable experience. So, sit back, relax, and let Math Magicians do the magic for you!</p>
  </div>
);

export default Home;
