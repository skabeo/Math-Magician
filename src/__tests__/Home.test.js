import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from '../components/Home';

describe('Home component', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders correctly', () => {
    render(<Home />);
    expect(screen.getByText(/And that's not all/i)).toBeInTheDocument();
  });
});
