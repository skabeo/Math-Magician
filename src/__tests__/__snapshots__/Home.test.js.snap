// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="white homeCont"
  >
    <h2
      class="homeTitle"
    >
      Welcome to our page
    </h2>
    <p
      class="homePara"
    >
      Welcome to Math Magicians, the one-stop destination for all your math needs! Are you tired of staring at your calculator, trying to figure out basic calculations? Well, look no further because we have you covered! With Math Magicians, you can now perform basic calculations with just a few clicks.
    </p>
    <p
      class="homePara2"
    >
       And that's not all - we also have a surprise for you! How about some random quotes to keep you motivated and inspired throughout the day? Yes, you read that right! We've combined the power of math and inspiration to create an unforgettable experience. So, sit back, relax, and let Math Magicians do the magic for you!
    </p>
  </div>
</DocumentFragment>
`;
